src/transformers/generation_utils.py

```diff
@@ -691,6 +693,8 @@ def _get_logits_processor(
         exponential_decay_length_penalty: Tuple,
         logits_processor: Optional[LogitsProcessorList],
         renormalize_logits: Optional[bool],
+        supress_tokens: Optional[List[int]],
```

Comment:
Why is there no default here? This would avoid the RAG mess (also applies to all arguments before, I don't know why we have a method with 19 positional arguments and no defaults but it is a bit crazy)

Reply:
I totally agree, I followed the previous structure but will add the default now.

Reply:
Agree!

Reply:
Feel free to do in a follow-up PR though. Here maybe just add `None` to the two new args